internal/static/i18n/en.yaml

```
@@ -560,7 +560,7 @@ EventTypes:
     removed: Target deleted
   user:
     added: User added
-    selfregistered: User registered himself
+    selfregistered: User registered
```

As the action is `selfregistered` should we keep a portion of `self` in the translation. Like:

```suggestion
    selfregistered: User registered themself
```

BTW if your dictionary complains, like mine "themself" singular is perfectly valid: https://english.stackexchange.com/a/68767